### ISSUE TITLE
list component is not scrollable

### ISSUE THREAD

**Issue by Alexilator** [huntabyte/shadcn-svelte#256]
### Current Behavior

When creating a select component and it has more options than the height of the screen / select list can display, the option list is not scrollable, no scrollbar appears either.

### Expected Behavior

A select component with more options to display than can be displayed within the defined list height or screen height should be scrollable.

### Steps To Reproduce

1. create a project following the installation instructions for shadcn-svelte and add the select component
2. create a select component with a list of like 10 options on the `+page.svelte`
3. set the height for the select to something like h-32
4. start the dev server open the page and klick on the select component to display the list
5. try to scroll

### More Information

1. I found this when trying to use the select component in shadcn-svelte. Then reproduced similar behavior in a vanilla svelte project using bits-ui directly, see [issue 109 from bits-ui](https://github.com/huntabyte/bits-ui/issues/109)
2. I could not reproduce it in a project using melt-ui
3. using arrow keys works
4. there is some odd behavior when you jiggle the mouse pointer between the bottom of the list and the last visible option: it seems to jump to the next option

## snippet
```svelte
<script lang="ts">
	import * as Select from "$lib/components/ui/select";
    const array = Array.from({ length: 100 });
    
    </script>
    <Select.Root >
        <Select.Trigger class="w-[180px]">
          <Select.Value placeholder="Select an option" />
        </Select.Trigger>
        <Select.Content class=" h-32">
          <Select.Group>
            <Select.Label>Fruits</Select.Label>
            {#each array as option}
            <Select.Item value={option}
              >{option}</Select.Item
            >
            {/each}
          </Select.Group>
        </Select.Content>
        <Select.Input name="optionSelect" />
    </Select.Root>
```

**Comment by huntabyte** [comment]
You need to apply `overflow-y-auto` to the `<Select.Content>` as well.

**Comment by huntabyte** [comment]
Does this resolve your issue @Alexilator ?

**Comment by Alexilator** [comment]
Yes, thank you very much!! Any outlook on when the combobox component will be released?

**Comment by Skylli202** [comment]
Hi, 

Sorry for commenting on a closed issue, but this did help me in my project. @huntabyte this kind of things, do you just expect people who are using this project to know it ? Or would you accept PR to add that kind of things as hint/help/faq on the component documentation pages ?

**Comment by ReflextionsDev** [comment]
Yeah allowing documentation contribution would be great
